weather-api-gateway/pom.xml

```diff
@@ -14,8 +14,8 @@
         <maven.compiler.target>1.8</maven.compiler.target>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <!-- update this version when Camel update theirs -->
-        <version.io.quarkus>0.22.0</version.io.quarkus>
-        <version.org.apache.camel.quarkus>0.2.0</version.org.apache.camel.quarkus>
+        <version.io.quarkus>1.0.0.CR1</version.io.quarkus>
```

Comment:
version 1.0.0.Final is released already.

Reply:
But Camel hasn’t updated their side yet :(